operator/internal/manifests/distributor.go

```diff
@@ -15,12 +15,14 @@ import (
 )
 
 const (
-	walVolumeName     = "wal"
-	configVolumeName  = "config"
-	storageVolumeName = "storage"
-	walDirectory      = "/tmp/wal"
-	dataDirectory     = "/tmp/loki"
-	secretDirectory   = "/etc/proxy/secrets"
+	walVolumeName          = "wal"
+	configVolumeName       = "config"
+	rulesStorageVolumeName = "rules"
+	storageVolumeName      = "storage"
+	walDirectory           = "/tmp/wal"
+	dataDirectory          = "/tmp/loki"
+	rulesStorageDirectory  = "/tmp/rules"
+	secretDirectory        = "/etc/proxy/secrets"
```

Comment:
Does it make sense to group this section by "component"? (for example putting the two `rules` constants together)

I also would not have expected this section to be in `distributor.go`, if I am honest :grinning:

Reply:
You are right, this belongs into `var.go`, however I saw the other day that @Red-GV schema-migration PR did the right move. So keeping this here for now.